compiler/control/OMROptions.cpp

```diff
@@ -3403,6 +3403,47 @@ OMR::Options::processOptionSet(
    return options;
    }
 
+/**
+ * Custom comparison function to compare two options.
+ * Used for STL binary search.
+ *
+ * @param a The first option.
+ * @param b The second option.
+ */
+
+bool
+OMR::Options::compareOptionsForBinarySearch(const TR::OptionTable &a, const TR::OptionTable &b)
+   {
+   int32_t lengthOfTableEntry;
+
+   const TR::OptionTable *optionToFind, *optionInTable;
+
+   // Check which option is the one passed in (optionToFind). We want to match
+   // only up to the number of characters of the option in the table, not the one
+   // passed in.
+   if (a.msg == "optionToFind")
+      {
+      optionToFind = &a;
+      optionInTable = &b;
+      }
+   else
+      {
+      optionToFind = &b;
+      optionInTable = &a;
+      }
+
+   lengthOfTableEntry = optionInTable->length;
+
+   // Ensure 'log@lf' is the same as 'log=lf' (Z language environment transform)
+   if (optionInTable->name[lengthOfTableEntry - 1] == '=' &&
+       optionToFind->length >= optionInTable->length &&
+       optionToFind->name[lengthOfTableEntry - 1] == '@' &&
+       strnicmp_ignore_locale(a.name, b.name, lengthOfTableEntry - 1) == 0)
```

Comment:
@fjeremic Is this transform still required? (IIRC, it was only needed for JCL, and I don't know if we still support JCL)